### DISCUSSION TITLE
Copy and paste rather than copy or paste? Huh.

### DISCUSSION THREAD

**Discussion by michaelwhotoon** [keiwando/toonsquid#133] [Q&A]
Hi Guys

Got a mountain. Cut it in half using the rectangle selection tool. Want to copy the selected half, then delete it then go to a new layer and paste copied half mountain on to that layer. However I am confronted with copy and paste or cut and paste menu selection. Tried fiddling and got no joy. I fluked it once but don't know how I did it. What is the procedure? Logic says there should be a copy menu selection and a paste menu selection. Its like saying stop/go when you want someone to stop lol.

Thanks Legends.

**Comment by wpd2012** [comment]
I need Copy and Paste as separate so I can do it on the next frame/layer
